### PR TITLE
Resolve imports

### PR DESCRIPTION
**Pull request by ElijahAhianyo**

### CODE REVIEW

File: docs/library/datadisplay/list.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 components:
-    - rx.radix.themes.list.item
-    - rx.radix.themes.list.ordered
-    - rx.radix.themes.list.unordered
+    - rx.radix.list.item
```
Comment (picklelo) [inline docs/library/datadisplay/list.md:3]: Can we event just do `rx.list.item` here?

**Review by picklelo — APPROVED** [review]
Looks good - wondering if we should remove the `radix` also and just have the `rx.` in here.

File: docs/library/datadisplay/list.md (hunk 1 above)
Comment (ElijahAhianyo) [reply docs/library/datadisplay/list.md:3]: Yes, we could. I left them in the radix namespace so its explicit that it's a radix component. But I could change them to use the `rx.` namespace if it looks problematic

Comment (picklelo) [reply docs/library/datadisplay/list.md:3]: We can do in a follow up if needed

**Pull request merged**